tailwind.config.js

```diff
@@ -0,0 +1,12 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: [
+    "./index.html",
+    "./src/**/*.{vue,js,ts,jsx,tsx}",
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
+
```

Comment:
I think we should avoid using tailwind, since the project is already using Uno.

Reply:
Personally, my vscode tailwind extensions did not work without this. Perhaps there is another extension for uno css